packages/palette/src/elements/Cards/MediumCard.tsx

```diff
@@ -1,10 +1,50 @@
 import React from "react"
+import { Box } from "../Box"
+import { Flex } from "../Flex"
+import { Image } from "../Image"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { CardTag } from "./CardTag"
 import { MediumCardProps } from "./MediumCard.shared"
 
 /**
  * `MediumCard` is a card with one image one tall image, and text for title and subtitle
  * at the bottom.
  */
-export const MediumCard: React.FC<MediumCardProps> = ({}) => {
-  return null
+export const MediumCard: React.FC<MediumCardProps> = ({
+  image,
+  title,
+  subtitle,
+  tag
+}) => {
+  return (
+    <Box width="280px" height="370px" position="relative">
+      <Flex
+        width="100%"
+        height="100%"
+        position="absolute"
+        top="0"
+        borderRadius="4px"
+        overflow="hidden"
+      >
+        <Image src={image} alt={title} height="100%" width="auto" />
+        <Box
+          style={{
+            background: "linear-gradient(rgba(0, 0, 0, 0), rgba(0,0,0,1))",
+            opacity: 0.15
+          }}
```

Comment:
We do this gradient overlay thing in quite a few places, seems like we might want to abstract that?

Reply:
That sounds good, but maybe out of scope for this PR? Happy to follow up this PR after VR launch

Reply:
Can one of you add this in an issue or front end practice or wherever the right place would be? 🙏